src/override/malloc.cc

```diff
@@ -116,6 +116,36 @@ extern "C"
   }
 #endif
 
+#if !defined(__NetBSD__)
+  SNMALLOC_EXPORT int
+    SNMALLOC_NAME_MANGLE(reallocarr)(void* ptr, size_t nmemb, size_t size)
```

Comment:
I was sure this signature must be wrong.  Surely, it should be `void**`, but I read the docs and your right.